Project.toml

```
@@ -52,6 +55,7 @@ DocStringExtensions = "0.8, 0.9"
 Enzyme = "0.11.14"
 FFTW = "1"
 Glob = "1.3"
+GPUArrays = "0.9.1"
```

did you mean to write

```suggestion
GPUArrays = "9.1.0"
```

---

My dyslexia is showing